kf/kf_unix.go

```diff
@@ -87,6 +87,14 @@ func (b *BPF) SetPrLimits() error {
 	return nil
 }
 
+// ProcessTerminate - Send sigterm to the process
+func (b *BPF) ProcessTerminate() error {
+	if err := syscall.Kill(b.Cmd.Process.Pid, syscall.SIGTERM); err != nil {
```

Comment:
can you do `b.Cmd.Process.Signal(syscall.SIGTERM)` or will that only work for SIGINT and SIGKILL?